equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/Crm4Controller.java

```diff
@@ -32,6 +32,7 @@ public ResponseEntity<Crm4FormDTO> getApplicationCrm4(Long usn, String profileAc
         );
         Crm4Model crmFormData = crmFileService.getCrmFormData(crmFormDetailsCriteriaDTO);
         crmFormData.getFormDetails().setAllQuotes();
+        crmFormData.addFileKeyToFurtherInfo(crmFormData.getFormDetails().getFurtherInformationModel().getAttachments());
```

Comment:
potential `NPE - Null Pointer Exception` - possible to do null check?
`crmFormData.getFormDetails()` May be null  OR
`crmFormData.getFormDetails().getFurtherInformationModel() `May be null

Reply:
good spot!! I think by this stage, if formDetails is null, it would had failed xml validation. And furtherNotificaitonModel should not be null by this point.. could be empty but not null. But we should make sure this is checked at some point.

Reply:
I just confirmed. this section is added to the sanity check. So we should be covered